pkgs/by-name/ka/karp/package.nix

```
@@ -0,0 +1,69 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitLab,
+  cmake,
+  qt6,
+  kdePackages,
+  ghostscript,
+  qpdf,
+}:
+
+stdenv.mkDerivation {
+  pname = "karp";
+  version = "0-unstable-2024-11-20";
+
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "graphics";
+    repo = "karp";
+    rev = "f26d6c43adc2feb2b0569df126f3a7be5d95ac2e";
+    hash = "sha256-w1wrPaqQ6NBtbY5OEtxGlc72mXuLrlefq6A02U9wWHc=";
+  };
+
+  nativeBuildInputs = [
+    cmake
```

```suggestion
    cmake
    ninja
```
Use ninja over make may speed up build times a bit